packages/myst-cli/src/session/session.ts

```diff
@@ -23,21 +23,23 @@ export function logUpdateAvailable({
   current,
   latest,
   upgradeCommand,
+  pypiUpgradeCommand,
   twitter,
 }: {
   current: string;
   latest: string;
   upgradeCommand: string;
+  pypiUpgradeCommand?: string;
   twitter: string;
 }) {
   return boxen(
     `Update available! ${chalk.dim(`v${current}`)} ≫ ${chalk.green.bold(
       `v${latest}`,
-    )}\n\nRun \`${chalk.cyanBright.bold(
-      upgradeCommand,
-    )}\` to update.\n\nFollow ${chalk.yellowBright(
-      `@${twitter}`,
-    )} for updates!\nhttps://twitter.com/${twitter}`,
+    )}\n\nRun \`${chalk.cyanBright.bold(upgradeCommand)}\` to update.\n\n${
+      pypiUpgradeCommand
+        ? `(Or if using PyPI, \`${chalk.cyanBright.bold(pypiUpgradeCommand)}\`) \n\n`
```

Comment:
We probably want both of these present so we do not confuse PyPI users with npm installation instructions.

We probably could figure out which the user needs by parsing the result of `which('myst')` but that feels like maybe overkill for the first pass.